_shared/hooks/post_gen_project.py

```diff
@@ -144,12 +169,13 @@ def main():
         template_ignore_patterns.extend([
             "{{ cookiecutter.package_name }}/__init__.py",
             "{{ cookiecutter.package_name }}/app.py",
+            "{{ cookiecutter.package_name }}/models/__init__.py",
```

Comment:
The cookiecutter will generate a starter `models/__init__.py` file when creating a new project with Postgres enabled (it has to otherwise `make sure` will be failing in the new project) but adding `models/__init__.py` to `template_ignore_patterns` means that `make template` won't update that file: projects are free to edit it directly.